tests/Unit/NumericalAlgorithms/LinearOperators/Test_Divergence.cpp

```diff
@@ -166,6 +169,12 @@ void test_divergence_impl(
   const auto& expected =
       get<Tags::div<Flux1<DataType, Dim, Frame>>>(div_fluxes);
   CHECK_ITERABLE_CUSTOM_APPROX(expected, div_vector, local_approx);
+
+  // Test logical divergence
+  auto logical_fluxes =
+      transform::first_index_to_different_frame(fluxes, inv_jacobian);
+  logical_divergence(make_not_null(&div_fluxes), logical_fluxes, mesh);
```

Comment:
I would create a `logical_div_fluxes` variable since the current code I think would pass if `logical_div_fluxes` was a no-op